tests/adapters/test_remote_agent.py

```
@@ -0,0 +1,156 @@
+        {
+            "config": config,
+            "runtime_context": context.to_mapping(),
+            "base_dir": str(ROOT),
+        }
+    )
+    assert runtime._endpoint == f"{api_server}/v1{path}"
+
+    result = await runtime.invoke(AgentRunRequest(input="Hello."), context)
+    await runtime.stop()
+
+    captured = requests.get(f"{api_server}/_requests", timeout=5).json()
```

Ruff (0.16.2)

[warning] Async functions should not call blocking HTTP methods

(ASYNC210)